lib/taiko.js

```diff
@@ -638,6 +638,23 @@ module.exports.deleteCookies = async (cookieName, options = {}) => {
   }
 };
 
+/**
+ * Resize the browser window
+ *
+ * @example
+ * await resizeWindow({width:600, height:800})
+ *
+ * @returns {Promise<void>}
+ */
+module.exports.resizeWindow = async (options = {}) => {
+  validate();
+  if (options.height === undefined || options.width === undefined) {
+    throw new Error('No height and width provided');
```

Comment:
Noticed this when testing
```
Version: 1.1.3 (Chromium: 88.0.4314.0)
Type .api for help and .exit to quit

> openBrowser()
>  ✔ Browser opened
> resizeWindow({ width: 300 });
 ✘ Error: No height and width provided, run `.trace` for more info.
```

To make the message generic I think the following is better

```suggestion
    throw new Error('Please specify the window height and width');
```